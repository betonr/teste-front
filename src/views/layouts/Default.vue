<template>
    <div class="general">
        <m-header></m-header>

        <!-- CONTENT PAGE -->
        <router-view/>

        <m-footer></m-footer>        
    </div>
</template>

<script>
import Header from '@/views/components/default/Header'
import Footer from '@/views/components/default/Footer'

export default {
    components: {
        'm-header': Header,
        'm-footer': Footer
    }
}
</script>

<style lang="sass" scoped>
</style>
