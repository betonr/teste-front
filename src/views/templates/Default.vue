<template>
    <div class="general">
        <m-header />

        <!-- CONTENT PAGE -->
        <router-view />

        <m-footer />        
    </div>
</template>

<script>
import Header from '@/views/components/default/Header'
import Footer from '@/views/components/default/Footer'

export default {
    name: "Default",

    components: {
        'm-header': Header,
        'm-footer': Footer
    }
}
</script>

<style lang="sass" scoped>
</style>
