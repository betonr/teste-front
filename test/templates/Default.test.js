import Vue from 'vue'
import vueRouter from 'vue-router'

Vue.use(vueRouter)

import { shallowMount } from '@vue/test-utils'
import Default from '@/views/templates/Default.vue'

describe('Default.test.js', () => {
    
    let wrapper
    beforeAll(() => {
        wrapper = shallowMount(Default)
    })

    test('modification - snapshot', () => {
        expect(wrapper.element).toMatchSnapshot()
    })
    
})