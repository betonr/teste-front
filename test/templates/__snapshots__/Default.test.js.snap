// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default.test.js modification - snapshot 1`] = `
<div
  class="general"
>
  <m-header-stub />
   
  <router-view-stub
    name="default"
  />
   
  <m-footer-stub />
</div>
`;
